### PR TITLE
fix: align alias with webpack

### PR DESCRIPTION
**Pull request by ilteoood**
## Summary

Align alias with webpack ones

## Test Plan

Automatic tests

## Require Documentation?

- [ ] No
- [x] Yes, the corresponding rspack-website PR is https://github.com/web-infra-dev/rspack-website/pull/618

### CODE REVIEW

File: packages/rspack-cli/src/utils/options.ts [hunk 1]
```diff
@@ -36,7 +37,8 @@ export const commonOptions = (yargs: yargs.Argv) => {
 			devtool: {
 				type: "boolean",
 				default: false,
-				describe: "devtool"
+				describe: "devtool",
+				alias: "d"
```
Comment (h-a-n-a) [inline packages/rspack-cli/src/utils/options.ts:41]: This one should be tested as wel ;-)

File: package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
     "url": "https://github.com/web-infra-dev/rspack"
   },
   "engines": {
-    "pnpm": "8.14.3"
+    "pnpm": ">=8.14.3"
```
Comment (h-a-n-a) [inline package.json:44]: Lockfile may change in future major versions. Let'me revert this for you ;-)
```suggestion
    "pnpm": "8.14.3"
```

**Comment by h-a-n-a** [comment]
Thank you!

**Review by h-a-n-a — DISMISSED** [review]

**Comment by ilteoood** [comment]
@h-a-n-a the PR related to the documentation has been already merged. What about this?

**Comment by ilteoood** [comment]
Ping @h-a-n-a

**Comment by ilteoood** [comment]
Ping @h-a-n-a @JSerFeng

**Review by JSerFeng — APPROVED** [review]

**Comment by JSerFeng** [comment]
Thank you !

**Pull request merged**